website/docs/d/acm_certificate.html.markdown

```diff
@@ -6,7 +6,7 @@ description: |-
   Get information on a Amazon Certificate Manager (ACM) Certificate
 ---
 
-# aws_acm_certificate
+# Data Source: aws\_acm\_certificate
```

Comment:
Can we remove all of the antislashes? An effort was made recently to clean them all so that we don't have to worry about them anymore!

Reply:
![psyduckmini](https://user-images.githubusercontent.com/61721/34182135-a16468ac-e4da-11e7-9ac8-6c604054318e.gif)